rp2040-hal/src/pio.rs

```diff
@@ -633,19 +689,33 @@ pub struct Rx<SM: ValidStateMachine> {
 }
 
 impl<SM: ValidStateMachine> Rx<SM> {
+    fn block(&self) -> &pac::pio0::RegisterBlock {
```

Comment:
I know we already have a `fn block` in pio.rs, but I don't like the name. It sounds too much like a command. (Block for something, like in `nb::block`.)
I'd prefer, for example, `fn register_block`.